### DISCUSSION TITLE
How translations from POEditor implements to the app?

### DISCUSSION THREAD

**Discussion by HafizBeyy** [streetcomplete/StreetComplete#6037] [Q&A]
i wanna help to translation but when i translate smt, im having hard time to understand the context. sometimes united words with underscores help me understand it but mostly its not. where translations used in the code or how? i hope i can understand and help better if i can see where are them used?

**Comment by matkoniecz** [comment]
These codes with underscores are used in app code. 

For example if use of say map_attribution_osm is unclear you can search at https://github.com/search?q=repo%3Astreetcomplete%2FStreetComplete%20map_attribution_osm&type=code which may or may not help.

In general uses in app code are in form `R.string.string_code_shown_in_translation_platform` - it will be replaced when app is run by actual translated words, depending on language being used.

There are also comments at POEditor - if you comment there I will get an email. I am trying to reply to such questions.

You can also create a discussion thread, like this one.

Using app also helps - you often can spot mistranslated terms or notice where some text is used.

(and BTW, thanks to all translators for help!)
